src/js/pages/jobs/JobsTable.js

```diff
@@ -0,0 +1,167 @@
+import {Link} from 'react-router';
+import React from 'react';
+import {Table} from 'reactjs-components';
+
+import JobTableHeaderLabels from '../../constants/JobTableHeaderLables';
+import ResourceTableUtil from '../../utils/ResourceTableUtil';
+import ServiceTableUtil from '../../utils/ServiceTableUtil';
+import TableUtil from '../../utils/TableUtil';
+import Tree from '../../structs/Tree';
+
+const StatusMapping = {
+  'Completed': 'text-mute',
+  'Running': 'text-success',
+  'Scheduled': 'text-mute',
+  'Unscheduled': 'text-mute'
+};
+
+const METHODS_TO_BIND = [
+  'renderHeadline',
+  'renderStatus',
+  'renderLastRunStatus'
+];
+
+class JobsTable extends React.Component {
+  constructor() {
+    super();
+
+    METHODS_TO_BIND.forEach((method) => {
+      this[method] = this[method].bind(this);
+    }, this);
+  }
+
+  getColGroup() {
+    return (
+      <colgroup>
+        <col style={{width: '25%'}} />
+        <col />
+        <col style={{width: '25%'}} />
+      </colgroup>
+    );
+  }
+
+  getColumns() {
+    let className = ResourceTableUtil.getClassName;
+    let heading = ResourceTableUtil.renderHeading(JobTableHeaderLabels);
+
+    return [
+      {
+        className,
+        headerClassName: className,
+        prop: 'name',
+        render: this.renderHeadline,
+        sortable: true,
+        sortFunction: ServiceTableUtil.propCompareFunctionFactory,
```

Comment:
I'd like to either use a dedicated `JobsTable` util or have a unified util that works for both... Same is true for the `ResourceTableUtil`. What do you think?